Formula/f3d.rb

```diff
@@ -0,0 +1,49 @@
+class F3d < Formula
+  desc "Fast and minimalist 3D viewer"
+  homepage "https://kitware.github.io/F3D/"
+  url "https://gitlab.kitware.com/f3d/f3d/-/archive/v1.0.1/f3d-v1.0.1.tar.gz"
+  sha256 "fb362dba3ccf49db9e8841d8a5310f37399bfea8866b6e040ce85670d54b97f7"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "vtk"
+
+  def install
+    # Is fixed upstream, remove this line for v1.1
+    inreplace "src/F3DOptions.cxx", "/etc/f3d/", "#{etc}/f3d/"
+
+    args = std_cmake_args + %W[
+      -DMACOSX_BUILD_BUNDLE:BOOL=OFF
+      -DBUILD_SHARED_LIBS:BOOL=ON
+      -DBUILD_TESTING:BOOL=OFF
+      -DF3D_INSTALL_DEFAULT_CONFIGURATION_FILE:BOOL=ON
+      -DCMAKE_INSTALL_NAME_DIR:STRING=#{lib}
+      -DCMAKE_INSTALL_RPATH:STRING=#{lib}
+    ]
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
+    end
+
+    mkdir_p etc/"f3d"
+    mv etc/"f3d/config.json", etc/"f3d/config.json.old" if File.exist?(etc/"f3d/config.json")
```

Comment:
I'm confused by this. Is `config.json` supposed to be a user config?

If it is, then it is probably not a great experience to have your customised config be replaced by a template config every time there is a new version of f3d.

Reply:
No, it is the system config. The user config is located in `~/.config/f3d/config.json` which is not installed and must be created manually.  
While we recommend using user config, it can make sense to modify the system config in order to have the same config for all users though.  
Is there a way to ask the user if he wants to replace the installed config by the default config during the installation?  
One solution could be to check if a system config already exists in which case we do not replace it.

Reply:
Hmm. I see. I'm not sure this does what you want, though, as the variable `etc` resolves to `#{prefix}/Cellar/f3d/#{version}/etc`.

Reply:
Thanks, there was an issue upstream that I have fixed using `inreplace`. There is no need for file copies now.